logback-classic/src/main/java/ch/qos/logback/classic/disruptor/LogValueEvent.java

```diff
@@ -0,0 +1,46 @@
+package ch.qos.logback.classic.disruptor;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.spi.AppenderAttachableImpl;
+
+import com.lmax.disruptor.EventFactory;
+
+/**
+ * Disruptor Producers to consumers The content of the carrier.
+ * 
+ * @author Liu-Luke
+ */
+
+public class LogValueEvent {
+	public LogValueEvent() {
+
+	}
+	private ILoggingEvent eventObject;
+	private AppenderAttachableImpl<ILoggingEvent> parent;

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/tony19/logback/file/2395872186/issues/source?bid=3222618&fileBranchId=3222679#l19)